### DISCUSSION TITLE
How to improve quality of cover image

### DISCUSSION THREAD

**Discussion by ultraspy** [hadynz/obsidian-kindle-plugin#259] [Q&A]
No matter what I do in the template, my image URL consistently pulls up 160-sized images. I know I can get better image quality by having the end of the URL direct to the correct size. I must be messing up the code, so I'd appreciate some input. Here's my most recent attempt:

`{% if imageUrl %}![|300]({{imageUrl}}){% endif %}`

Also, thanks in advance

**Comment by poitch** [comment]
imageUrl is pointing to the 160px version of the cover. The way I went about this was to replace the ._SY160 in the `imageURL` with nothing, which points to the highest quality possible. You can also replace the _SY160 with _SY1024 for a 1024px version of the cover.

```{% if imageUrl %}![|300]({{ imageUrl | replace("._SY160", ""} }){% endif %}```

**Reply by ultraspy** [reply]
It only took me a year to check back but THANK YOU!
